spec.bs

```diff
@@ -647,6 +647,33 @@ Each time a <{fencedframe}> navigates to a [=fenced frame config=], it is instan
 [=fenced frame config instance=], which governs the particular [=browsing context group=] inside the
 [=fenced navigable container/fenced navigable=].
 
+<h4 id=fenced-frame-config-use-cases>Use cases</h4>
+
+Rendering an ad created through an ad auction:
+
+An ad auction API runs an auction and determines a winning ad. Details about the winning ad must be
+hidden from the embedder, and the embedding context cannot be allowed to influence the environment
+of the <{fencedframe}>. Either of those would allow for information to flow across the fenced frame
+boundary, which can allow for colluding parties to join cross-site data and build a profile on the
+user. To prevent that, the ad auction API constructs a [=fenced frame config=] whose underlying
+[=fenced frame config/mapped url|URL=] is opaque to the embedding context. The [=fenced frame
+config=] is also constructed with restrictions on what the [=fenced frame config/container size=]
+and [=fenced frame config/content size=] of the frame must be and what the [=fenced frame
+config/effective enabled permissions|permissions policy=] of the frame must be, as those can be used
+as fingerprinting vectors.
+
+Displaying a personalized payment button:
+
+An e-commerce site embeds a <{fencedframe}> that has a "Pay now" button. At first, the {{Document}}
+hosted in the <{fencedframe}> has no 1p cookie/storage access, so information can freely flow in and
+out without risk of cross-site data joining. Because of that, the fenced frame can be constructed
+directly from the web platform using the {{FencedFrameConfig}} constructor without compromising
+privacy. The button at this point has no personalized data in it since it can't access the credit
+card data yet. The {{Document}} can only read that credit card data once it turns off all network
```

Comment:
"the credit card data" is pretty confusing here. This is the first time "credit card" is used in this section without any introduction...I'd have expected you to call out that as a prerequisite, you can imagine some personalized credit card data associated with the first-party storage of the fenced frame's origin.... _and then_ go into the fact that by default the FF cannot read this data.